src/service/light.rs

```diff
@@ -301,9 +301,9 @@ impl CIEColor {
     /// # Example
     ///
     /// ```
+    /// use hues::service::CIEColor;
     /// let hex = "#FAA020";
     /// let cie = CIEColor::from_hex(hex).unwrap();
-    /// assert_eq!(CIEColor { x: 0.0, y: 0.0 }, cie);
```

Comment:
I will look later for correct numbers from a trusted source to use for testing here.